README.rst

```diff
@@ -416,13 +416,18 @@ get_endpoint utility to do so:
 
     # '/buckets/payments/collections/receipts/records/c6894b2c-1856-11e6-9415-3c970ede22b0'
 
-Handling YAML date objects
---------------------------
+
+Handling datetime and date objects
+----------------------------------
 
 JSON doesn't have a specific type cast for dates while YAML does.
```

Comment:
Maybe change this to something like `In addition to the data types supported by JSON, kinto-http.py also supports native Python date and datetime objects.`